ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/QueryFeaturesWithArcadeExpression/README.md

```diff
@@ -0,0 +1,49 @@
+# Query features with arcade expression
+
+Query features on a map using an arcade expression.
+
+![](screenshot.png)
+
+## Use case
+Arcade is a portable, lightweight, and secure expression language used to create custom content in ArcGIS applications. Like other expression languages, it can perform mathematical calculations, manipulate text, and evaluate logical statements. It also supports multi-statement expressions, variables, and flow control statements. What makes Arcade particularly unique when compared to other expression and scripting languages is its inclusion of feature and geometry data types. This sample uses an arcade expression to query the number of crimes in a neighborhood in the last 60 days.
+
+## How to use the sample
+Click on any neighborhood to see the number of crimes in the last 60 days in a callout.
+
+## How it works
+1. Create a `PortalItem` using the URL and ID.
+2. Create a `Map` using the portal item.
+3. Set the visibility of all the layers to false, except for the layer named "RPD Beats  - City_Beats_Border_1128-4500".
+4. Set up a lambda or a listener for clicks on the map.
```

Comment:
Consider rephrasing. We don't need to mention the lambda. Other samples use the following language:

* Connect to the `MouseClicked` event on the `MapView`.
* Set the map to a `MapView` and connect to the `MapQuickView::mouseClicked` signal.
* Listen for `mouseClicked` signal on the map view.
* Create a connection to `MapQuickView::mouseClicked`.

etc

Reply:
Different SDKs handle and verbalize this this differently, for example Android sets up a listener. I will simplify the wording to avoid mentioning lambdas.